docs/assistant/security-assistant.asciidoc

```diff
@@ -2,4 +2,97 @@
++
+* *Self-managed (on-premises) deployments*: Add the feature flag to the `kibana.yml` file, which is used to {kibana-ref}/settings.html[configure {kib}], then restart {kib}.
+* *{ecloud} deployments*: Use the YAML editor in the {ecloud} console to add the feature flag to {cloud}/ec-manage-kibana-settings.html[{kib} user settings].
+
+. Create a {kibana-ref}/gen-ai-action-type.html[Generative AI connector], using the AI provider's API key and URL to configure communication between {elastic-sec} and the provider. You can do this in {kib} at *Stack Management* -> *Connectors*, or from within the Security Assistant.
++
+NOTE: The Generative AI connector type requires the `assistantEnabled` feature flag, and won't be available without it.
+
+[discrete]
+[[start-chatting]]
+== Start chatting
+
+To open the Security Assistant, press *Cmd + ;* (or *Ctrl + ;* in Windows) from anywhere in the {security-app}. This opens the *Welcome* conversation context, where you can ask general questions about {elastic-sec}.
+
+You can also chat with the Security Assistant from several areas of {elastic-sec}, and context-specific data and prompts will populate your conversation.
+
+* <<view-alert-details, Alert details>> or Event details flyout: Click *Chat* while viewing the details of an alert or event.
+* <<rules-ui-management, Rules page>>: Select one or more rules, then click the magic wand icon (🪄✨) at the top of the page next to the *Rules* title.
+* <<data-quality-dash, Data Quality dashboard>>: Select the *Incompatible fields* tab, then click *Chat*. (This is only available for fields marked red, indicating they're incompatible).
+* <<timelines-ui, Timeline>>: Select the *Security Assistant* tab.
+
+All chat history persists within your session, so you can navigate away to other areas of {elastic-sec}, then later return to ongoing conversations.
+
+[discrete]
+[[interact-with-assistant]]
+== Interact with the Security Assistant
+
+Use these features to adjust and act on your conversations with the Security Assistant:
+
+* Select a _system prompt_ at the beginning of a conversation to establish how detailed and technical you want the Security Assistant's answers to be.
++
+[role="screenshot"]
+image::images/system-prompt.gif[The system prompt drop-down menu,90%]
++
+NOTE: The system prompt is only configurable at the start of a conversation. To reconfigure, clear the chat and start a new conversation.
+
+* Select a _quick prompt_ at the bottom of the chat window to get help writing a prompt for a specific purpose, such as summarizing an alert or converting a query from a legacy SIEM to {elastic-sec}. Available quick prompts vary based on context. You can also add your own custom quick prompts for questions that you frequently ask the Security Assistant.
++
+[role="screenshot"]
+image::images/quick-prompts.png[Quick prompts highlighted below a conversation,90%]
+
+* Use available buttons to perform actions in the conversation history and prompt entry area:
```

Comment:
Is there a certain scenario where these buttons wouldn't be available? Also, would "these" or "the following" work in place of "available" or is that too literal? 

```suggestion
* Use these buttons to perform actions in the conversation history and prompt entry area:
```